compiler/rustc_const_eval/src/interpret/operand.rs

```diff
@@ -750,7 +771,7 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
                         }
                         untagged_variant
                     }
-                    Ok(tag_bits) => {
+                    (_, Ok(tag_bits)) => {
```

Comment:
```suggestion
                    (true, Ok(tag_bits)) => {
```
explicit is better than implicit